libraries/botbuilder-dialogs-adaptive/src/input/confirmInput.ts

```diff
@@ -38,6 +38,14 @@ export class ConfirmInput extends DialogCommand implements DialogDependencies {
     /**
      * Activity to send the user.
      */
+    public set prompt(value: string|Partial<Activity>) {
```

Comment:
This was needed because the original implementation was constructor based, but we need property based setting of this to allow declarative patterns